tools/pipelines/pipeline.html.js

```diff
@@ -37,7 +37,7 @@ module.exports = function setupHTMLPipeline(gulp) {
       return file.contents.toString();
 
     } else if (ext === '.js') {
-      if (args.env === 'production') {
+      if (process.env.NODE_ENV === 'production') {
```

Comment:
question(non-blocking): 

Does this not already pick up `process.env.NODE_ENV`? 

I think, ultimately, this change _is_ needed, but I remember testing this a while back and finding that, when no `--env` flag was present, it _did_ pick up the Node ENV vars. Is that not the case? Mostly just curious 🤔 .

Reply:
huh interesting I didnt see it picking it up on my local server, do you remember how you tested?

Reply:
Nah it's been far too long, but if you're not seeing it, then I must have been mistaken.